src/OpenTelemetry.Exporter.Geneva/ExceptionStackExportMode.cs

```diff
@@ -14,9 +14,17 @@ public enum ExceptionStackExportMode
     Drop,
 
     /// <summary>
-    /// Exception stack traces are exported as strings.
+    /// Exception stack traces are exported as string, using a culture agnostic representation of the results of ToString() implementation of the Exception.
+    /// Though formatted in culture-agnostic way, this is more targetted towards human readability.
+    /// See <see href="https://learn.microsoft.com/dotnet/api/system.exception.tostring"/>.
```

Comment:
TODO: Mention that ToString() can be overridden by users, so this could return completely irrelevant information too!

Reply:
same with stackTrace too.